Tasks/NodeTool/nodetool.ts

```diff
@@ -131,8 +135,28 @@ async function acquireNode(version: string): Promise<string> {
 
     let downloadUrl = 'https://nodejs.org/dist/v' + version + '/' + urlFileName;
 
-    let downloadPath: string = await toolLib.downloadTool(downloadUrl);
+    let downloadPath: string;
+
+    try 
+    {
+        downloadPath = await toolLib.downloadTool(downloadUrl);
```

Comment:
download tool takes an optional destination file path, so the extra copy can be avoided.